torch_agents/cogment_verse_torch_agents/hive_adapter/training_run.py

```diff
@@ -276,15 +219,19 @@ async def run_trials(trial_configs, max_parallel_trials):
                     samples_generated += 1
 
                     with TRAINING_ADD_SAMPLE_TIME.time():
-                        model.consume_training_sample(sample.current_player_sample)
+                        model._replay_buffer.add(
```

Comment:
this should basically be a call to `update` no ?

Reply:
updated